src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceServiceTest.kt

```diff
@@ -142,6 +142,79 @@ class LicenceServiceTest {
     assertThat(licence.nomsId).isEqualTo(aLicenceEntity.nomsId)
     assertThat(licence.bookingId).isEqualTo(aLicenceEntity.bookingId)
     assertThat(licence.pnc).isEqualTo(aLicenceEntity.pnc)
+//    assertThat(licence.isInPssPeriod).isFalse()
+
+    verify(licenceRepository, times(1)).findById(1L)
+  }
+
+  @Test
+  fun `service transforms key fields of a licence should return isInPssPeriod false if LED is null`() {
```

Comment:
now you have the logic on licence you can add a `LicenceTest` that tests the functionality without having to be concerned about repositories or services